src/Pelagos/Util/GmlUtil.php

```diff
@@ -0,0 +1,54 @@
+<?php
+namespace Pelagos\Util;
+
+/**
+ * This is a utility class for dealing with GML manipulation.
+ */
+class GmlUtil
+{
+    
+    const GML_NAMESPACES = array(
+                    'xmlns:gml' => 'http://www.opengis.net/gml/3.2',
+                    'xmlns:xsi' => 'http://www.w3.org/2001/XMLSchema-instance',
+                    'xsi:schemaLocation' => 'http://www.opengis.net/gml/3.2 ' .
+                    'http://schemas.opengis.net/gml/3.2.1/gml.xsd'
+                );
+                
+    /**
+     * This function add namespace for validation to the given gml.
+     *
+     * @param string     $gml        Gml that needs namespace.
+     * @param array|null $namespaces Array of attributes and values, can be left blank for std namespaces.
+     *
+     * @return string GML string with namespace.
+     */
+    public static function addNamespace($gml, array $namespaces = self::GML_NAMESPACES)
+    {
+        if (($gml != null) and (1 == preg_match('/gml/', $gml))) {
```

Comment:
Type comparision

Reply:
You could use !== for the first one, but null does have type. Also preg_match can return a boolean, that would cover this case also. But I don't think it would return true, only false.